vllm/worker/vineyard_llm_cache.py

```diff
@@ -107,6 +131,20 @@ def _init_vineyard_logger(self):
         for handler in logger.handlers:
             vineyard.logger.addHandler(handler)
 
+    def _run_background_loop(self):
+        logger.info("VineyardKVCache background loop is running")
+        while True:
+            # Wait until there is a task in the queue
+            update_fn = self._update_tasks.get()
+            # Run the task
+            try:
+                update_fn()
+                logger.debug(
+                    f"Completed an update op, current task queue size={self._update_tasks.qsize()}"
+                )
+            except Exception:
```

Comment:
Could you please add some metrics to track the qps, latency, and payload size of the update calls?

Reply:
added metrics to track queuing delay, task execution time, and updated tokens. current metric mechanism cannot report qps.